### ISSUE TITLE
Fabric port

### ISSUE THREAD

**Issue by vos6434** [lilgallon/AimAssistanceMod#9]
Could we get a Fabric port? This mod would work great with the Midnight Controls mod.

**Comment by lilgallon** [comment]
Yes, it's almost done ! It is working at the moment, I will release it soon. The code is available here: https://github.com/lilgallon/AimAssistance

The new version is designed to be compatible with any control mod, and will be easier to update. Once release it will be available for download on this page https://www.curseforge.com/minecraft/mc-mods/aim-assistance-fabric

**Comment by lilgallon** [comment]
Here is the version (not tested): https://github.com/lilgallon/AimAssistance/releases/tag/1.0.0RC1-MC1.19.x

Requires:
- kotlin for fabric: https://www.curseforge.com/minecraft/mc-mods/fabric-language-kotlin/files/all?filter-game-version=1738749986%3a73407

Optional:
- Modmenu
- Cloth config (to change the mod config)

**Comment by vos6434** [comment]
Crashes
https://github.com/lilgallon/AimAssistance/issues/6
